external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -199,9 +199,9 @@ abstract class KafkaMicroBatchSourceSuiteBase extends KafkaSourceSuiteBase {
     val topic = newTopic()
     testUtils.createTopic(topic, partitions = 5)
 
-    testUtils.sendMessages(topic, (0 until 15).map(x => {
-      "foo-" + x
-    }).toArray, Some(0))
+    testUtils.sendMessages(topic, (0 until 15).map{ case x =>
```

Comment:
sorry for the nits: space between `p` and `{` / same happens in foreachBatch

Not sure style checker will check this, but it is good practice to run`dev/scalastyle` before pushing the change. Not all styles are checked with style checker unfortunately. 

Please go through https://github.com/databricks/scala-style-guide when you have some time.